### ISSUE TITLE
Unable to swipe back to previous page on iOS

### ISSUE THREAD

**Issue by KingCrows** [swup/swup#309]
Hi there,
Building a website and using swup for the page transitions. Only problem I'm running into, is that on iOS (and maybe even Android, don't know...) i'm unable to go back to the previous page with the back arrow of with a swipe gesture. 

Already set animateHistoryBrowsing to false, but problem keeps occurring...

Any tips to fix this? Because this really is a bad UX thing, so need to fix this before going live!
Thanks!!

**Comment by daun** [comment]
Does any of your other code use the History API or update URL hashes? Can you provide an isolated CodePen example or similar?

**Comment by KingCrows** [comment]
As far as is know, the History API or update URL cases are not used.
Here is a snippet of the init for swup

`if (document.getElementById('swup')) {
    var swupScroll = new SwupScrollPlugin({
        doScrollingRightAway: false,
        animateScroll: false,
    });

    const options = {
        linkSelector: 'a[data-swup]',
        cache: true,
        animateHistoryBrowsing: false,
        plugins: [
            swupScroll,
            new SwupJsPlugin(jsOptions),
        ],
    };

    const swup = new Swup(options);

    swup.on('contentReplaced', init);

    swup.on('clickLink', () => {
        document.body.classList.remove('is-menu-open');
        document.body.classList.remove('over-hidden')
        document.getElementById('cursor').classList.remove('cursor-big');
    });
}
`

**Comment by KingCrows** [comment]
Oh btw, the same issue occurs in Safari browser on MacBook Pro...

**Comment by daun** [comment]
Looks about right. Hard to say what the matter is without an isolated example that reproduces the issue you're seeing. I'm happy to take a look in Safari if you provide one.

**Comment by KingCrows** [comment]
I've send the url to your e-mailaddress that's in you profile

**Comment by daun** [comment]
Seems like a server issue. Your server is sending `406 Not Acceptable` headers for ajax requests. Swup then cancels the visit and starts a normal page load. In your case you don't notice because all pages start out in red even on initial load. If you get your server to send a `200 OK` header it should work fine.

**Comment by KingCrows** [comment]
Thanks for your quick reply!! But in the Chrome browser a `200 OK` header is send? And only in Safari it isn’t?

**Comment by daun** [comment]
It did look like both browsers were cancelling the swup visits and forcing a normal page load. Only difference seems to be how they handle that in regards to the browser history. You'll have to dig a bit deeper from here.

**Comment by KingCrows** [comment]
I looked into this and added some code to my htaccess:

```
<IfModule mod_security.c>
SecFilterEngineOff
SecFilterScanPOSTOff
</IfModule>
```

No sure this really does anything. Problem keeps occurring. And on every page I see a `200 OK` in my console. Anything else where I can see those 406 headers or another thing I should look into?

**Comment by KingCrows** [comment]
Maybe any tips on how to fix this? My hosting service tells me it’s an issue with the website and nothing on their side...

**Comment by daun** [comment]
Not really, no. This seems to be a backend issue. I'd ask for help on whatever forums or chats there are for your backend framework.

**Comment by KingCrows** [comment]
Maybe a specific type of `AddHandler` or `AddType` I can add to my htaccess?

**Comment by gmrchk** [comment]
It's clear that swup is not an issue and there is some problem with a server/backend setup. The only thing I can add is that swup sets a header `X-Requested-With` to values `swup`, which could be mishandled as it's not a usual value for the header. I'm afraid we can't help any more than that with the server setup. 

If you do figure out what was the problem, please post it here for reference.

**Comment by KingCrows** [comment]
After trying some things like changing the `Header` in my htaccess and also asked my hosting of they knew a solution or saw something in logs on their side. But all of this didn't help, unfortunately! 

Then I began some backwards debugging. My setup was all `PHP` files, where files like `head.php`, `header.php`, `footer.php` etc. are included in the page PHP files. Nothing special I think and just a normal generic setup for a website.

I then included all my includes in the PHP pages, not with `required_once` or something, but just copy-paste the code from these files in my page PHP file. That didn't help. Eventually I made all my PHP pages, `HTML` files, with no includes and just everything is in the `index.html`, `about-us.html` etc. and this seems to fix the problem for me... Luckily it's not a huge website, so not working with includes will work for this time :-)

So that fixed it for my. Probably just a problem met PHP, includes and my hosting, I guess.
